blocks/library/table/editor.scss

```diff
@@ -1,11 +1,30 @@
 .wp-block-table {
+	table {
+		border-collapse: collapse;
+		width: 100%;
+	}
+
 	td,
 	th {
 		padding: 0.5em;
 		border: 1px solid currentColor;
 	}
 
-	td[data-mce-selected="1"], th[data-mce-selected="1"] {
-		background-color: $light-gray-500;
+	td[data-mce-selected="1"],
+	th[data-mce-selected="1"] {
+		background-color: $light-gray-300;
 	}
 }
+
+// Style the resize handles
+.ephox-snooker-resizer-rows {
```

Comment:
Those names look alien, is there any way to make them look as WordPress internals? :)

Reply:
I'm afraid those are from deep within the core of the TinyMCE library, so we can't change them unless we change those core files.

Reply:
Sounds okey, I had to double check :)